src/hammer-vlsi/technology/nangate45/nangate45.tech.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "Nangate45 Library",
```

Comment:
Is this https://www.silvaco.com/products/nangate/FreePDK45_Open_Cell_Library/ ?

Reply:
no, https://github.com/The-OpenROAD-Project/OpenROAD-flow/tree/master/flow/platforms/nangate45

it has some extra collateral.